source/server/server.cc

```diff
@@ -412,6 +414,25 @@ void InstanceImpl::initialize(const Options& options,
   heap_shrinker_ =
       std::make_unique<Memory::HeapShrinker>(*dispatcher_, *overload_manager_, stats_store_);
 
+  for (const auto& bootstrap_extension : bootstrap_.bootstrap_extensions()) {
+    auto& factory = Config::Utility::getAndCheckFactory<Configuration::BootstrapExtensionFactory>(
+        bootstrap_extension);
+    auto config = Config::Utility::translateAnyToFactoryConfig(
+        bootstrap_extension.typed_config(), messageValidationContext().staticValidationVisitor(),
+        factory);
+    bootstrap_extensions_.push_back(
+        factory.createBootstrapExtension(*config, serverFactoryContext()));
+  }
+
+  if (!bootstrap_.default_socket_interface().empty()) {
```

Comment:
nit
```suggestion
  if (bootstrap_.has_default_socket_interface()) {
```

Reply:
Done!

Reply:
Just recompiled and finally remembered why I didn't do it like that. It turns out "has_default_socket_interface()" does not exist. Could it be because it's a string?

Reply:
ah sorry my bad, it doesn't exist in proto3 if it's a string. nvm